app/controllers/users_controller.rb

```diff
@@ -82,8 +81,8 @@ def new_registration
   def create_registration
     @user = current_user
     @user.assign_attributes(user_params)
-    @user.user_attributes[:department] = params[:user][:user_attributes][:department]
-    @user.user_attributes[:school] = params[:user][:user_attributes][:school]
+    @user.department = params[:user][:user_attributes][:department]
+    @user.school = params[:user][:user_attributes][:school]
     @user.user_attributes[:marli_renewal] = params[:user][:user_attributes][:marli_renewal]
     @user.user_attributes[:affiliation_text] = params[:user][:user_attributes][:affiliation_text]
```

Comment:
@hab278 If you just set these to database fields instead of user_attributes, it should continue to function the same. Make sure you do a global search for user_attributes so there are no vestiges.